lib/cdo/contact_rollups.rb

```diff
@@ -494,8 +494,8 @@ def self.update_roles
     add_role_from_script_sections_taught("CSF Teacher", CSF_SCRIPT_LIST)
     # CSD and CSP scripts are mapped to course - identify CSD/CSP teachers
     # that way
-    add_role_from_course_sections_taught("CSD Teacher", "csd-2017")
-    add_role_from_course_sections_taught("CSP Teacher", "csp-2017")
+    add_role_from_course_sections_taught("CSD Teacher", "csd-2018")
+    add_role_from_course_sections_taught("CSP Teacher", "csp-2018")
```

Comment:
Same thing here.  It looks like we added the `-2017` as [part of introducing course versions](https://github.com/code-dot-org/code-dot-org/pull/22519) so there may not be a precedent for whether an addition or replacement is appropriate here.  From my current mental model of contact rollups, we're assigning 'roles' to users here, and it's unclear to me whether a "CSD Teacher" role implies that they're currently teaching CSD, or just means they have ever taught CSD.

@davidsbailey do you know? Or should we ask someone from marketing how they'd like these roles assigned?

Reply:
Sorry, @islemaster , I don't know what the intended behavior of contact rollups is or how it's used. This is a question for someone familiar with contact rollups.

Reply:
@sureshc can you give some context on this?

Reply:
I spoke with Clare in person yesterday about this.  Contact Rollups gathers all data, past and present, about people who have opted in to receive emails from us.  That means we need to continue querying old script versions, while adding queries for new course versions.